src/Neo.VM/VMCatchableException.cs

```diff
@@ -13,10 +13,12 @@
 
 namespace Neo.VM
 {
-    public class CatchableException : Exception
+    public class VMCatchableException : Exception
     {
-        public CatchableException(string message) : base(message)
+        public VMCatchableException(string message) : base(message)
         {
+            if (string.IsNullOrEmpty(message))
+                throw new ArgumentException("Message cannot be null or empty.", nameof(message));
```

Comment:
I prefer to avoid this, this error will be throw during runtime, is not a warning for us

Reply:
yes remove this, and add a default message when `null`.